src/main/kotlin/com/kotlindiscord/database/migrator/MigrationsManager.kt

```diff
@@ -0,0 +1,126 @@
+package com.kotlindiscord.database.migrator
+
+import org.jetbrains.exposed.dao.LongEntity
+import org.jetbrains.exposed.dao.LongEntityClass
+import org.jetbrains.exposed.dao.id.EntityID
+import org.jetbrains.exposed.dao.id.IdTable
+import org.jetbrains.exposed.sql.*
+import org.jetbrains.exposed.sql.transactions.transaction
+import org.reflections8.Reflections
+
+object Migrations : IdTable<Long>() {
+    override val id = long("id").entityId()
+    val applied = bool("applied").default(false)
+    override val primaryKey = PrimaryKey(id, name = "PK_MIGRATION_ID")
+}
+
+class Migration(id: EntityID<Long>) : LongEntity(id) {
+    companion object : LongEntityClass<Migration>(Migrations)
+
+    var applied by Migrations.applied
+}
+
+/**
+ * Interfaces with the database to manage and apply migrations. Requires a Database connection.
+ */
+class MigrationsManager {
+
+    /**
+     * Gets the id of the migration which was most recently applied to the database.
+     */
+    private val currentMigrationId: Long?
+        get() {
+            var maxMigrationId: Long? = null
+            transaction {
```

Comment:
Separating this from the line above is probably preferable for readability (with a blank line, I mean)